### PR TITLE
Refactor the conflicts episode; closes #80

### LINKED ISSUE #80: Suggestions: conflicts
- [x] some of our feedback was asking for more practice on conflicts.  This will probably happen tomorrow, but we could have more exercises here.  Once I made a tar.gz with a repo which you could just unpack and 'git pull' and simulate the conflict.  But clone + merge from the clone works just as well.
- [x] Acknowledge at top that conflicts are scary, but not that bad if you stop and take a deep breath.  Also they are luckily rare.
- [x] Manual resolution paragraph: don't forget to run "git diff" after you have attempted resolution.  There are also a lot of bullet point here - can it be organized some more?
- [x] Resolution using mergetool - at this point we had already resolved so couldn't use it.  Should make another exercise to use this, or show how to reset the successful resolution and then they can try again (may be too complex).
- [x] "theirs" and "ours" strategy: I've never used this before, but seems useful.  Again consider how to let them try this themselves.  And personal note, I guess that the checkout resets all changes to theirs, not just the conflicts?
- [x] The exercise is much later than we we actually resolved it - in class we did it during the "manual method" steps and then had nothing else to do for the rest of the lesson. 
- [x] Avoiding conflicts: discus google drive resolution strategy (very small delta-t and delta-chars) and choosing arbitrarily.  Push early and often - the sooner you merge and push, the less likelyhood you need to do it (others will have to).  But if everyone uses this strategy, then everyone benefits!  Also this paragraph can be broken up some - technical measures, human measures, collaboration measures.  Maybe mention of types of problems there might be - e.g. semantic conflicts that merge but don't work.  Importance of talking!  
- [x] Don't be afraid of git because of conflicts.  You don't get them using other systems because you simply can't do the kinds of things you do in git.  You can take human measures to reduce them.

### PR DESCRIPTION
**Pull request by bast**
Work in progress ...

### CODE REVIEW

File: _episodes/10-conflicts.md [hunk 1]
```diff
@@ -21,68 +21,122 @@ Then a merge commit appears (unless fast-forward) without you even noticing.
...
+But sometimes the same line or portion of the code/text is modified on two branches and Git issues a conflict.
 Then you need to tell Git which version to keep.
 
 There are several ways to do that as we will see.
 
+Please remember:
+
+- Conflicts look scary, but not that bad if you stop and take a deep breath. Also they are luckily rare.
+- Don't be afraid of Git because of conflicts. You may not meet some conflicts using other systems because you simply can't do the kinds of things you do in Git.
+  You can take human measures to reduce them.
```
Comment (rkdarst) [inline _episodes/10-conflicts.md]: `-` in last line?

File: _episodes/10-conflicts.md [hunk 2]
```diff
@@ -211,53 +295,59 @@ To configure Git to avoid creating backups at all:
...
+- Collaboration measures
+  - Conflicts can be avoided if you think and talk with your colleagues before committing.
+  - Semantic conflicts that merge but don't work: Importance of talking!
+- Project layout measures
+  - Fortran people: modifying common blocks often causes conflicts.
+  - Modifying global data often causes conflicts.
+  - Monolithic entangled spaghetti-code maximizes risk of conflicts.
+  - Modular programming minimizes risk of conflicts.
+- Technical measures
+  - Push early and often.
```
Comment (rkdarst) [inline _episodes/10-conflicts.md]: Maybe bold here.  I don't know if this should be written in the exercise, but when presenting conflicts I say: this is one of the happy, rare circumstances when everyone doing the selfish thing (pushing as early as practical) results in best case for everyone!

File: _episodes/10-conflicts.md [hunk 3]
```diff
@@ -211,53 +295,59 @@ To configure Git to avoid creating backups at all:
...
+  - One branch for one task only.
+- Collaboration measures
+  - Conflicts can be avoided if you think and talk with your colleagues before committing.
+  - Semantic conflicts that merge but don't work: Importance of talking!
+- Project layout measures
+  - Fortran people: modifying common blocks often causes conflicts.
+  - Modifying global data often causes conflicts.
+  - Monolithic entangled spaghetti-code maximizes risk of conflicts.
+  - Modular programming minimizes risk of conflicts.
+- Technical measures
```
Comment (rkdarst) [inline _episodes/10-conflicts.md]: Oh, what about keeping up to date with upstream / pulling often?  I have often begun working, and pull when I am about to push with predictable results...  This should also be bold.

**Comment by rkdarst** [comment]
It looks very nice, simple, non-intimidating, and to the point.  Very good!

**Comment by bast** [comment]
Thank you for the comments! I will patch these issues and then merge.

**Pull request merged**